tensorflow/docs_src/tutorials/recurrent.md

```diff
@@ -51,11 +51,25 @@ The core of the model consists of an LSTM cell that processes one word at a
 time and computes probabilities of the possible values for the next word in the
 sentence. The memory state of the network is initialized with a vector of zeros
 and gets updated after reading each word. For computational reasons, we will
-process data in mini-batches of size `batch_size`.
+process data in mini-batches of size `batch_size`.  In this example, it is important to note that `current_batch_of_words` does not correspond to a "sentence" of words.  Every word in a batch should correspond to time t.  Tensorflow will automatically sum the gradients of each batch for you.
+
+For example:
+```
+ t=0  t=1  t=2     t=3
+[The, fox, is,     quick]
+[The, fox, jumped, high]
+
+words_in_dataset[0] = [The, The]
+words_in_dataset[1] = [fox, fox]
+words_in_dataset[2] = [is, jumped]
+words_in_dataset[3] = [quick, high]
+num_batches = 4, batch_size = 2
```

Comment:
I can add another variable describing the number of timesteps (time_steps = 4). So that was initially my source of confusion when looking at the tutorial, the time steps don't necessarily apply in the first example (non-truncated BP). It only applies when we split it up into batches. Otherwise, we would just process one word at at time over the entire dataset. Does that make sense?